pkg/vault/test/foundry/BalancerContractRegistry.t.sol

```diff
@@ -213,6 +213,14 @@ contract BalancerContractRegistryTest is BaseVaultTest {
         registry.registerBalancerContract(ContractType.POOL_FACTORY, DEFAULT_NAME, ANY_ADDRESS);
     }
 
+    function testIsTrustedRouter() public {
+        vm.prank(admin);
+        registry.registerBalancerContract(ContractType.ROUTER, DEFAULT_NAME, ANY_ADDRESS);
+
+        assertTrue(registry.isActiveBalancerContract(ContractType.ROUTER, ANY_ADDRESS), "ANY_ADDRESS is not a Router");
+        assertTrue(registry.isTrustedRouter(ANY_ADDRESS), "ANY_ADDRESS is not a Trusted router");
+    }
```

Comment:
It'd be nice to test a deregister, in case we want to retire a router.

Reply:
There are several tests for deregistration; since `isTrustedRouter` is just a wrapper, I don't think we need to duplicate it for that